app/controllers/availabilities_controller.rb

```diff
@@ -20,8 +21,58 @@ def update_availabilities
     redirect_to availabilities_path
   end
 
+  def default_index
+    @time_ranges = TimeRange.order(:start_time)
+    @timeslots = Hash[Timeslot.joins(:time_range)
+                              .map do |timeslot|
+                        [[timeslot.day, timeslot.time_range_id], timeslot]
+                      end
+    ]
+  end
+
+  def default_edit; end
+
+  def default_update; end
+
+  def all
```

Comment:
We normally use a verb here (after all it's a controller action). Maybe `show_everyone`?